apps/envited-x-data-space/common/asset/types.ts

```diff
@@ -33,77 +34,53 @@ export interface Manifest {
   '@id': string
   '@type': string
   'manifest:hasManifestReference': ManifestLink
-  'manifest:hasLicense': ManifestLink
+  'manifest:hasLicense': ManifestLicense
   'manifest:hasArtifacts': ManifestArtifacts
   'manifest:hasReferencedArtifacts': []
 }
 
 export interface ManifestContext {
-  xsd: string
-  gx: string
-  skos: string
-  sh: string
-  manifest: string
-  envited: string
-  hdmap: string
-  rdf: string
-}
-
-export interface ManifestHasAccessRole {
-  '@type': string
-  '@id': AccessRole
-}
-
-export interface ManifestHasCategory {
+  'xsd': string
+  'gx': string
+  'skos': string
+  'sh': string
+  'manifest': string
+  'envited-x': string
+  'hdmap': string
+  'rdf': string
```

Comment:
I think we miss a lot of context types here if we check SCHEMAS, do we need to define them?
<img width="331" alt="Scherm­afbeelding 2025-03-27 om 12 37 22" src="https://github.com/user-attachments/assets/7bd06acf-f241-4e20-a4c5-e1d35417938a" />